File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        n = len(nums)
        count = 0
        for i in range(n):
            if(nums[i] < nums[i-1]):
                count += 1
        if(count <= 1):
            return True
        return False

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int count = 0;
        int n = nums.length;
        for(int i=0;i<n;i++){
            if(nums[(i-1+n)%n] > nums[i]){
                count++;
            }
        }
        return count<=1;
    }
}